pkg/scaffold/v2/prometheus/monitor.go

```diff
@@ -22,23 +22,23 @@ import (
 	"sigs.k8s.io/kubebuilder/pkg/scaffold/input"
 )
 
-var _ input.File = &PrometheusServiceMonitor{}
+var _ input.File = &ServiceMonitor{}
 
-// PrometheusMetricsService scaffolds an issuer CR and a certificate CR
-type PrometheusServiceMonitor struct {
+// ServiceMonitor scaffolds an issuer CR and a certificate CR
+type ServiceMonitor struct {
 	input.Input
 }
 
 // GetInput implements input.File
-func (f *PrometheusServiceMonitor) GetInput() (input.Input, error) {
-	if f.Path == "" {
-		f.Path = filepath.Join("config", "prometheus", "monitor.yaml")
+func (p *ServiceMonitor) GetInput() (input.Input, error) {
+	if p.Path == "" {
+		p.Path = filepath.Join("config", "prometheus", "monitor.yaml")
 	}
-	f.TemplateBody = monitorTemplate
-	return f.Input, nil
+	p.TemplateBody = serviceMonitorTemplate
+	return p.Input, nil
```

Comment:
Leave this as `f` that comes from `file`

Reply:
it makes no diff. However, done 👍

Reply:
One of the most recent PR was to unify all them to `f` so that every file use it instead of mixing more than 6 different letters. It was to avoid having `GetInput` with `f` and `Validate` with `g` for example, as that is a golint error.